main/modes/mode_gamepad.c

```diff
@@ -484,6 +630,34 @@ void gamepadMainLoop(int64_t elapsedUs __attribute__((unused)))
 
         // Draw touch strip
         int16_t tBarX = gamepad->disp->w - TOUCHBAR_WIDTH;
+
+        // If we're on the generic gamepad and touch analog is enabled, plot the extra indicator on the screen
+        if(gamepad->gamepadType == GAMEPAD_GENERIC && gamepadToggleSettings->touchAnalogOn)
+        {
+            int32_t center, intensity;
+            if(gamepad->gpState.buttons & ((touchMap[0] | touchMap[1] | touchMap[2] | touchMap[3] | touchMap[4])))
+            {
+                getTouchCentroid(&center, &intensity);
+                // Subtract 2 from TOUCHBAR_WIDTH while scaling so we can draw a 3px-wide cursor centered on center, without covering the box borders
+                center = (TOUCHBAR_WIDTH - 2) * center / 1024 + 1;
+            }
+            else
+            {
+                center = 1;
```

Comment:
This isn't consistent with the reported value to the PC. When not touched, the reported value is 50%, not 0%. You can verify this with the Windows Game Controller Properties dialog.

Reply:
Ahh this was throwing me off https://github.com/AEFeinstein/Super-2023-Swadge-FW/blob/main/main/modes/mode_gamepad.c#L792